IshiHisashi/GlucoFit
# feat: JDEV-37 - Create low level components (Buttons, Chips)

## Description of this change
- create reusable component for button/chip
- add some svg icons
  - our icons are coming from unicons library (https://github.com/Iconscout/unicons?tab=readme-ov-file) and it has React Native version, but we can't just install it to use because React version that is bundled with the library is too old (v16) to be compatible with our app. To be safe, I thnk we should manually download necessary icons from our UI kit and create jsx element using this tool: https://react-svgr.com/playground/?native=true&typescript=true
- this is not fixed yet as the designers are developing components right now. We can enhance this later. 

## Steps to test
1. `cd frontend`, and run the app on an emulator
2. You can check example usages on Insights screen. 

<img src="https://github.com/user-attachments/assets/48b3209a-ac91-4b06-96ff-a091010d32b2" width="150px" />

## Ticket link
https://team-jupiter-capstone.atlassian.net/browse/JDEV-37
